SepulcherOfTheFirstOnes/Locales/esES.lua

```diff
@@ -13,25 +13,25 @@ end
 
 L = BigWigs:NewBossLocale("Artificer Xy'mox v2", "esES")
 if L then
-	--L.traps = "Traps" -- Stasis Trap
-	--L.sparknova = "Sparknova" -- Hyperlight Sparknova
-	--L.relocation = "Tank Bomb" -- Glyph of Relocation
-	--L.relocation_count = "%s S%d (%d)" -- Tank Bomb S1 (1) // Tank Bomb (stage)(count)
-	--L.wormholes = "Wormholes" -- Interdimensional Wormholes
-	--L.wormhole = "Wormhole" -- Interdimensional Wormhole
-	--L.rings = "Rings S%d" -- Rings S1 // Forerunner Rings Stage 1/2/3/4
+	--L.traps = "Trampa de estasis" -- Stasis Trap
+	--L.sparknova = "Chispanova de hiperluz" -- Hyperlight Sparknova
+	--L.relocation = "Bomba de tank" -- Glyph of Relocation
+	--L.relocation_count = "%s S%d (%d)" -- Bomba de tank S1 (1) // Bomba de tank (stage)(count)
```

Comment:
Anything after a `--` is a comment. It won't load. There is no need to translate it.
It's just to help you understand what you're translating.
For example `L.traps = "Traps" -- Stasis Trap`
Here after the comment `--` we say `Stasis Trap` because that's the full name of the spell, to help you understand that "Traps" is just a shorter word.

So changing it to `-- Bomba de tank S1 (1)` is not necessary, it can stay as English.